apps/web/src/components/layout/components/SideNav.tsx

```diff
@@ -165,6 +174,13 @@ export function SideNav({}: Props) {
   );
 }
 
+const CloseButtonStyled = styled(CloseButton)`
+  position: absolute;
+  top: 7px;
+  z-index: 2;
+  right: 10px;
+`;
+
```

Comment:
```suggestion
const CloseButtonStyled = styled(CloseButton)`
  position: absolute;
  top: 7px;
  z-index: 2;
  right: 10px;
`;

const CloseButtonStyled = createPolymorphicComponent<'button', CloseButtonProps>(_CloseButtonStyled);

```

Reply:
Yes, but for someone like me who has used Material UI extensively, this workaround seems a bit more verbose, so not a fan 😅

Reply:
yeah seems kind of weird :)